CONTRIBUTING.md

```diff
@@ -1,5 +1,9 @@
 # Contributing
 
+### Supported Node.js versions
+
+All the current LTS versions are supported.
```

Comment:
I was generic here, do you think it's enough or we want to be explicit @simonecorsi ?

Reply:
I think this should be good enough! So when LTS changes we don't have to update the document! 🙏